src/saveDocs.ts

```diff
@@ -58,6 +59,18 @@ async function addMetaData(pkg: RawPkg): Promise<FinalPkg> {
     },
   };
 
+  const keepAlternativeNames =
+    final.popular ||
+    (!final.isDeprecated &&
+      !final.isSecurityHeld &&
+      (final.downloadsLast30Days >=
+        config.alternativeNamesNpmDownloadsThreshold ||
+        final.jsDelivrHits >= config.alternativeNamesJsDelivrHitsThreshold));
+
+  if (!keepAlternativeNames) {
```

Comment:
```suggestion
  const hasFewDownloads =
    final.downloadsLast30Days <= config.alternativeNamesNpmDownloadsThreshold &&
    final.jsDelivrHits <= config.alternativeNamesJsDelivrHitsThreshold;

  const dropAlternativeNames =
    !final.popular &&
    (final.isDeprecated || final.isSecurityHeld || hasFewDownloads);

  if (dropAlternativeNames) {
```

this reads a little simpler to me, what do you think?